pkg/reconciler/route/resources/ingress.go

```diff
@@ -174,6 +189,70 @@ func makeACMEIngressPaths(challenges map[string]v1alpha1.HTTP01Challenge, domain
 }
 
 func makeIngressRule(domains []string, ns string, visibility netv1alpha1.IngressVisibility, targets traffic.RevisionTargets) *v1alpha1.IngressRule {
+	return &v1alpha1.IngressRule{
+		Hosts:      domains,
+		Visibility: visibility,
+		HTTP: &v1alpha1.HTTPIngressRuleValue{
+			Paths: []v1alpha1.HTTPIngressPath{
+				*makeBaseIngressPath(ns, targets),
+			},
+		},
+	}
+}
+
+func makeErrorPathForUndefinedTag() *v1alpha1.HTTPIngressPath {
+	return &v1alpha1.HTTPIngressPath{
+		Splits: []v1alpha1.IngressBackendSplit{{
+			IngressBackend: v1alpha1.IngressBackend{
+				ServiceNamespace: "not-existing-ns",
```

Comment:
This fails validation, which [requires all service namespaces to match the ingress namespace](https://github.com/knative/serving/blob/e14075aa8b4883c2d6d30c74c08e89ed356e9453/pkg/apis/networking/v1alpha1/ingress_validation.go#L143).